src/index.js

```diff
@@ -5,9 +5,14 @@ import { BrowserRouter } from 'react-router-dom'
 import store from './redux/store/configureReduxStore'
 import './index.css'
 import App from './App'
+import injectTapEventPlugin from 'react-tap-event-plugin'
 
 const target = document.getElementById('root')
 
+// Needed for onTouchTap
+// http://stackoverflow.com/a/34015469/988941
+injectTapEventPlugin()
```

Comment:
I assume tapping means mobile device tapping? I thought we were not supporting mobile right now? Not that is crazy to bring in, just want to understand the motivation behind it.

Reply:
From the Material UI docs: 
"Some components use react-tap-event-plugin to listen for touch events because onClick is not fast enough This dependency is temporary and will eventually go away. Until then, be sure to inject this plugin at the start of your app"

So instead of onClick, try using onTouchTap for Material UI components.

Reply:
Well thats interesting. Good find.